meetupselector/user/models.py

```diff
@@ -53,6 +63,7 @@ class User(AbstractBaseUser, PermissionsMixin):
     )
     date_joined: models.DateTimeField = models.DateTimeField(_("date_joined"), auto_now_add=True)
     is_staff: models.BooleanField = models.BooleanField(_("is_staff"), default=False)
+    is_superuser: models.BooleanField = models.BooleanField(_("is_superuser"), default=False)
```

Comment:
This field is not required because in `AbstrachBaseUser` you already have the superuser field.